ankaios_sdk/_components/workload_state.py

```diff
@@ -358,6 +358,20 @@ def get_filter_mask(self) -> str:
         return f"workloadStates.{self.agent_name}." \
                + f"{self.workload_name}.{self.workload_id}"
 
+    def to_proto(self) -> _ank_base.WorkloadInstanceName:
```

Comment:
This functions is called `_to_proto` for the other classes and also is used with the other name in [request.py:222](https://github.com/eclipse-ankaios/ank-sdk-python/pull/62/files#diff-a64e76a96ae947e596fdfcc393a3f87ef2c0ffb4187fa0c58ccdef36bd493d3bR222).
```suggestion
    def _to_proto(self) -> _ank_base.WorkloadInstanceName:
```